src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -89,6 +89,7 @@ export const SimplePowerBIReportEmbed = ({
   reports,
   reportConfiguration,
   scenario,
+  displayIfCurrentScenarioNotReady,
```

Comment:
Je pense que `currentScenarioNotReady` est suffisant, vu le nom des autres autres bool ?

Reply:
Après discussion, on peut partir sur `alwaysShowReports` en précisant dans la doc quelque chose comme `(placeholder will never be displayed, even if the simulation failed)`